### DISCUSSION TITLE
Using Stacking to make predictions

### DISCUSSION THREAD

**Discussion by xtendai** [sebp/scikit-survival#364] [Q&A]
Hello,

We are currently trying to use `sksurv.meta.Stacking` to combine different survival models to improve performance. 
Our current implementation is as follows:

```python
base_estimators = (('cox',CoxPHSurvivalAnalysis(), ('coxnet',CoxnetSurvivalAnalysis()), ('rsf',RandomSurvivalForest())
meta_estimator = RandomSurvivalForest()
meta = Stacking(meta_estimator, base_estimators)
meta.fit(X_train,y_train)
score = meta.score(X_test,y_test)
print(f"Score with Stacking: {score}")
meta.predict(X_test)
```

Is the above correct or are we missing something? The documentation states that the `fit()` method trains the base estimators.  Does the `meta_estimator` also get trained or does it need to be trained separately?  If so how. 
Also, while `meta.predict(X_test)` works, calling `meta_predict_proba(,,,)` or `meta.predict_survival_function(,..)`  gives an error (meta does not have the attribute...).    How do we use it to predict survival probabilites for various times? 
Any more information on this feature or examples of implementation would be very helpful. 
Thanks,

**Comment by sebp** [comment]
The meta-estimator will be trained too.

Currently, `Stacking` does not support predict `predict_survival_function`, because not all estimators can predict survival functions (e.g. Survival SVM). But I agree that if the meta-estimator supports it, so should `Stacking`.
